src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -0,0 +1,599 @@
+---
+layout: post
+title: Fast playback with video preload
+authors:
```

Comment:
```suggestion
subhead: How to accelerate your media playback by actively preloading resources.
authors:
```
This page seems a bit naked without a subheading:

![image](https://user-images.githubusercontent.com/4713486/96778609-c5d99c00-13a0-11eb-9081-e304596bea2a.png)